src/components/Tabs.vue

```diff
@@ -72,7 +73,10 @@
                 return this.tabs.find(tab => tab.hash === hash);
             },
 
-            selectTab(selectedTabHash) {
+            selectTab(selectedTabHash, event) {
+                // see if we should store the hash in the url fragment
+                if (event && !this.options.useUrlFragment) event.preventDefault();
```

Comment:
I don't like no blocked `if`. Could you use `{` `}` here?

Reply:
yes, fixed.